bot/constants.py

```diff
@@ -5,6 +5,9 @@
 QUOTES_CHANNEL_ID = 463657120441696256
 QUOTES_BOT_ID = 292953664492929025
 LOGGING_CHANNEL_ID = 538494690601992212
+HQ_ID = 532654287386443777
```

Comment:
These are listed in the "Fun" constants section. Maybe they should have a comment at the top of them (and a blank line) to make it clear what their purpose is.

e.g.
```suggestion

# Channels for discussion about Game
HQ_ID = 532654287386443777
```